kroxylicious-kms-provider-aws-kms/src/main/java/io/kroxylicious/kms/provider/aws/kms/config/JdkTls.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.aws.kms.config;
```

Comment:
Probably not for this PR but should TLS settings/config really be standard thing that comes from the proxy itself rather than being a per plug-in concern?

Reply:
Yes, it should.  I *think* we want to share those classes amongst KMS implementation and potentially amongst Filter implementations.   I definitely want to get rid of the duplication but would like to think about this separate from this PR.